### DISCUSSION TITLE
serverless documentation cannot see the previous tutorial

### DISCUSSION THREAD

**Discussion by yeukfei02** [serverless/serverless#11033] [General]
https://www.serverless.com/framework/docs/getting-started

this link now does not contain the previous documentation, anyone can help to fix it?

thanks.

**Comment by mnapoli** [comment]
Hi @yeukfei02, could you clarify what is missing?

**Reply by yeukfei02** [reply]
like the previous page about serverless.yml, events, those pages?

**Reply by mnapoli** [reply]
Oh is it because we reorganized the menu?

Check out the "Usage" section: https://www.serverless.com/framework/docs/providers/aws/guide/events

**Reply by mnapoli** [reply]
All link should still exist in the menu, they should just be easier to find now.

**Reply by yeukfei02** [reply]
https://www.serverless.com/cloud/docs

like sls cloud, left side have the sidebar..i can clearly know what I want to search

btw can change back sls doc like sls cloud doc? thanks.

**Reply by mnapoli** [reply]
I don't understand what you are saying sorry. Could you explain more what you mean?

**Reply by yeukfei02** [reply]
<img width="1496" alt="Screenshot 2022-05-05 at 11 10 48 PM" src="https://user-images.githubusercontent.com/7151923/166954822-61f325de-e1d8-4b6d-902f-54273e6b6c40.png">
<img width="1496" alt="Screenshot 2022-05-05 at 11 10 52 PM" src="https://user-images.githubusercontent.com/7151923/166954844-9cf503ec-4acc-4777-810f-8e7c1b1de58b.png">


left side navbar.. u can see sls doc doesnot have..but sls cloud doc have

**Reply by mnapoli** [reply]
Ohhhh this is weird! The menu should show.

Could you try refreshing the page and doing a hard refresh too? I have no idea what's going on here 🤔 
Could you also check out the web developer tools to see if there are JS errors in the console?

**Reply by yeukfei02** [reply]
<img width="1496" alt="Screenshot 2022-05-05 at 11 18 28 PM" src="https://user-images.githubusercontent.com/7151923/166956313-64f0b336-8972-4fe4-b034-70bab5e64e4f.png">
i see some cors issue...

**Reply by yeukfei02** [reply]
<img width="1496" alt="Screenshot 2022-05-05 at 11 19 13 PM" src="https://user-images.githubusercontent.com/7151923/166956558-28a7cccd-6062-436c-95ac-c7fd0aa7f285.png">
this is sls cloud doc...

**Reply by mnapoli** [reply]
It feels like a cache issue, that's very weird. Have you tried a hard refresh? Or checking "Disable the cache" in the networks tab?

**Reply by yeukfei02** [reply]
> It feels like a cache issue, that's very weird. Have you tried a hard refresh? Or checking "Disable the cache" in the networks tab?

i think it is not related to cache..i tried in chrome and safari also same..the left side bar did not show

**Reply by mnapoli** [reply]
@yeukfei02 could you please try again? (and try clearing the cache too) I widened the CORS settings to allow any domain, maybe an extension or proxy is causing issues here 🤔 

Hopefully that resolves the issue 🤞

**Reply by yeukfei02** [reply]
ya now able to see now thanks!

**Reply by mnapoli** [reply]
Thanks for confirming 🎉

**Comment by yeukfei02** [comment]
i see `smooth-scrollbar.css ` 404 and https://assets.serverless-extras.com/website/framework/docs/menu.json net::ERR_FAILED 200

**Comment by yeukfei02** [comment]
@mnapoli 

it looks like the serverless documentation left side bar disappear again..can u help to check?

cors problem?

thanks

```
Access to fetch at 'https://assets.serverless-extras.com/website/framework/docs/menu.json' from origin 'https://www.serverless.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
```

**Reply by mnapoli** [reply]
@yeukfei02 oh very sorry about that! Let's try to debug the problem. Could you confirm that you are using Chrome? Or another browser?

If you run this command in the terminal, can you paste the output (I want to check that it contains `access-control-allow-*` headers):

```
curl -H "origin: https://serverless.com" --head https://assets.serverless-extras.com/website/framework/docs/menu.json
```

Could you also try, by any chance, to clear your browser cache one last time?

**Reply by yeukfei02** [reply]
i use chrome

output:

```
HTTP/2 200
content-type: application/json
content-length: 24339
date: Mon, 30 May 2022 13:50:58 GMT
access-control-allow-origin: *
access-control-allow-methods: GET, HEAD
last-modified: Thu, 26 May 2022 14:55:35 GMT
etag: "909858555314251156746a593ed8e87b"
cache-control: max-age=60
accept-ranges: bytes
server: AmazonS3
x-cache: Miss from cloudfront
via: 1.1 09c63a510feb1b96fe87d2cfe41d34ae.cloudfront.net (CloudFront)
x-amz-cf-pop: SIN2-P2
x-amz-cf-id: qjJCFhC9_an8lOCIpK7sQU8hjCu94OZj6RieNCFSSssA-8j95D5Xng==
```

**Reply by yeukfei02** [reply]
i try in firefox, chrome, safari also same..left side bar cannot see

**Reply by mnapoli** [reply]
@yeukfei02 could you try again? I've added a new CloudFront configuration to force the CORS headers to be sent no matter what. Hopefully this is the last of it 😅

**Reply by yeukfei02** [reply]
it is working..thanks
